dbux-projects/src/practiceSession/PracticeSessionState.js

```diff
@@ -4,9 +4,18 @@ import Enum from '@dbux/common/src/util/Enum';
 let PracticeSessionState = {
   Activating: 1,
   Solving: 2,
-  Solved: 3
+  Found: 3,
+  Solved: 4
 };
 
 PracticeSessionState = new Enum(PracticeSessionState);
 
-export default PracticeSessionState;
\ No newline at end of file
+export default PracticeSessionState;
+
+const FoundedTypes = new Array(PracticeSessionState.getValueMaxIndex()).map(() => false);
+FoundedTypes[PracticeSessionState.Found] = true;
+FoundedTypes[PracticeSessionState.Solved] = true;
+
+export function isStateFoundedType(traceType) {
```

Comment:
The name `isStateFoundedType` needs improvement:

* `found`, not `founded`
* Why are there two nouns in this? It seems to indicate that `state` and `type` are the same thing? And if so, why are there two names for it? :D